rskj-core/src/main/java/co/rsk/mine/MinerServerImpl.java

```diff
@@ -156,8 +156,10 @@ public void submitBitcoinBlock(String blockHashForMergedMining, co.rsk.bitcoinj.
             Block workingBlock = blocksWaitingforPoW.get(key);
 
             if (workingBlock == null) {
-                logger.warn("Cannot publish block, could not find hash " + blockHashForMergedMining + " in the cache");
-                return;
+                String message = "Cannot publish block, could not find hash {} in the cache" + blockHashForMergedMining + " in the cache";
```

Comment:
Weird message. It was OK

"Cannot publish block, could not find hash " + blockHashForMergedMining + " in the cache"

Reply:
Fixed